dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -426,24 +470,43 @@ public synchronized void initialize(Service service) {
                 }
             }
             for (DOMSource r : schemas) {
+                if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
+                   LOG.fine("initialize: Processing DOMSource node: " + 
+				(r != null ? StaxUtils.toString(r.getNode()) : "null"));
+                } // Liberty change end
                 if (bi.contains(r)) {
                     String ns = ((Document)r.getNode()).getDocumentElement().getAttribute("targetNamespace");
+                    if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
```

Comment:
Maybe continue using isLoggableFine instead of checking again?

Reply:
yes, will check boolean